changelog/fix_remove_backref_related_code_from.md

```diff
@@ -0,0 +1 @@
+* [#9177](https://github.com/rubocop-hq/rubocop/pull/9177): Remove back-ref related code from SpecialGlobalVars. ([@r7kamura][])
```

Comment:
This is a nitpick. Can you enclose the cop name in "`"?

```suggestion
* [#9177](https://github.com/rubocop-hq/rubocop/pull/9177): Remove back-ref related code from `Style/SpecialGlobalVars`. ([@r7kamura][])
```

Reply:
Nice catch! I rewrote the Changelog entry, the commit message, and the PR title with backticks and its full cop name 📝